scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -153,8 +153,21 @@ class FossId(
     private val secretKeys = listOf("serverUrl", "apiKey", "user")
 
     private val service: FossIdRestService
+
+    /**
+     * Scanner option: If this optional filter is set, only packages having an identifier in given namespace will be
+     * scanned.
+     */
     private val packageNamespaceFilter: String
```

Comment:
Other scanner implementations (at least ScanCode) list the configuration properties they support in the class comment. This is quite nice because you have an overview over the options available at a central place.